### PR TITLE
[DOCS] Describe how to use Elastic Agent to monitor Logstash

### PR DESCRIPTION
**Pull request by dedemorton**
## Release notes

[rn:skip]

## What does this PR do?

Adds Elastic Agent as another way to collect monitoring data.

This work is tracked by https://github.com/elastic/observability-docs/issues/2602.

There are additional PRs to address changes required to monitoring docs for other stack components. These are tactical fixes to get the agent info into all the docs.

Please respond to questions addressed to reviewers.

## Why is it important/What is the impact to the user?
Users need to know that Elastic Agent integrations are now available and how to use them.

## Checklist

- [x] My code follows the style guidelines of this project

## Related issues

- Relates https://github.com/elastic/observability-docs/issues/2602

### To Do before merging

- [x] Remove questions to reviewers.

### CODE REVIEW

File: docs/static/monitoring/monitoring-ea.asciidoc [hunk 1]
```diff
@@ -0,0 +1,100 @@
+[[monitoring-with-elastic-agent]]
+=== Collect {ls} monitoring data with {agent}
+[subs="attributes"]
+++++
+<titleabbrev>{agent} collection</titleabbrev>
+++++
+
+You can use {agent} to collect data about {ls} and ship it to the monitoring
+cluster. The benefit of {agent} collection is that the monitoring agent remains
+active even if the {ls} instance does not. Plus you can mange all your
```
Comment (klacabane) [inline docs/static/monitoring/monitoring-ea.asciidoc:10]:
```suggestion
active even if the {ls} instance does not. Plus you can manage all your
```

File: docs/static/monitoring/monitoring-ea.asciidoc [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+****
+**Note to reviewers:**
+
+I will update the role shown here after finalizing https://github.com/elastic/elasticsearch/pull/94348
+
+****
+
+To collect {ls} monitoring data, add a {ls} integration to an {agent} and
+deploy it to the host where {kib} is running.
```
Comment (klacabane) [inline docs/static/monitoring/monitoring-ea.asciidoc:48]:
```suggestion
deploy it to the host where {ls} is running.
```

File: docs/static/monitoring/monitoring-ea.asciidoc [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+****
+**Question for reviewers:**
+
+Will the Logstash integration be GA for 8.7.0? If not, I will add a bit of text
+to explain the filter options at the bottom of the page.
```
Comment (klacabane) [inline docs/static/monitoring/monitoring-ea.asciidoc:54]: Yes similar to Kibana we'll release a GA version soon

File: docs/static/monitoring/monitoring-ea.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/static/monitoring/monitoring-ea.asciidoc:10]: Ha! Good catch. This is my go-to typo. I'm sure there are still docs at Big Blue that talk about a "Metadata Manger"

**Comment by dedemorton** [comment]
@klacabane When the doc build finishes, this PR will be ready for final review. Thanks!

File: docs/static/monitoring/monitoring-overview.asciidoc [hunk 4]
```diff
@@ -16,6 +16,12 @@ For an introduction to monitoring your Elastic stack, see
 Make sure monitoring is enabled on your {es} cluster. Then configure *one* of
```
Comment (karenzone) [inline docs/static/monitoring/monitoring-overview.asciidoc:16]:
```suggestion
Make sure that monitoring is enabled on your {es} cluster. Then configure *one* of
```

File: docs/static/monitoring/monitoring-ea.asciidoc [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+points to your {ls} host URLs. By default, the integration collects {ls}
+monitoring metrics from `localhost:9600`. If that host and port number are not
+correct, update the `hosts` setting. If you configured {ls} to use encrypted
+communications, you must access it via HTTPS. For example, use a `hosts` setting
+like
+`https://localhost:9600`.
++
+**Elastic security.** If the Elastic {security-features} are enabled, expand
+**Advanced options** and enter the username and password of a user that has
+the `remote_monitoring_collector` role.
```
Comment (karenzone) [inline docs/static/monitoring/monitoring-ea.asciidoc:71]:
```suggestion
**Elastic security.** The Elastic {security-features} are enabled by default. 
Expand **Advanced options** and enter the username and password of a user that has
the `remote_monitoring_collector` role.
```

Comment (karenzone) [reply docs/static/monitoring/monitoring-ea.asciidoc:71]: Minor tweak to emphasize that security is important and is on by default.

If you like this change, please add it to the monitoring-mb.asciidoc (on or near line 118).

**Review by karenzone — APPROVED** [review]
Thank you for your work on this. Looks great!
I left some comments inline for your consideration. Otherwise, LGTM.

**Review by klacabane — APPROVED** [review]
🚀

**Comment by dedemorton** [comment]
@logstashmachine backport 8.7

**Comment by karenzone** [comment]
@logstashmachine backport 8.7

**Pull request merged**